wallet-new/docs/updates.md

```diff
@@ -0,0 +1,563 @@
+          produces:
+            - application/octet-stream
+          responses:
+            '204':
+              description: 'The download has started.'
+      /api/v1/update/skip:
+        put:
+          description: Skip the available update.
+          produces:
+            - application/json;charset=utf-8
+          responses:
+            '204':
+              description: 'The update has been skipped.'
+    definitions:
+      CCoin:
+        required:
+          - getCCoin
+        properties:
+          getCCoin:
+            type: string
+        type: object
+      WalletSoftwareUpdate:
+        required:
+          - wsuSoftwareVersion
+          - wsuBlockVesion
+          - wsuScriptVersion
+          - wsuImplicit
+          - wsuVotesFor
+          - wsuVotesAgainst
+          - wsuPositiveStake
+          - wsuNegativeStake
+          - wsuSeverity
+          - wsuUpdateState
+          - wsuSha256Checksum
+        properties:
+          wsuSoftwareVersion:
+            $ref: '#/definitions/SoftwareVersion'
+          wsuBlockVesion:
+            $ref: '#/definitions/BlockVersion'
+          wsuScriptVersion:
+            maximum: 65535
+            minimum: 0
+            type: integer
+          wsuImplicit:
+            type: boolean
+          wsuVotesFor:
+            maximum: 9223372036854776000
+            minimum: -9223372036854776000
+            type: integer
+          wsuVotesAgainst:
+            maximum: 9223372036854776000
+            minimum: -9223372036854776000
+            type: integer
+          wsuPositiveStake:
+            $ref: '#/definitions/CCoin'
+          wsuNegativeStake:
+            $ref: '#/definitions/CCoin'
+          wsuSeverity:
+            $ref: '#/definitions/UpdateSeverity'
+          wsuUpdateState:
+            $ref: '#/definitions/UpdateState'
+          wsuSha256Checksum:
+            description: "A checksum for data integrity."
+            type: string
+        type: object
+      SoftwareVersion:
+        required:
+          - svAppName
+          - svNumber
+        properties:
+          svAppName:
+            $ref: '#/definitions/ApplicationName'
+          svNumber:
+            maximum: 4294967295
+            minimum: 0
+            type: integer
+        type: object
+      ApplicationName:
+        required:
+          - getApplicationName
+        properties:
+          getApplicationName:
+            type: string
+        type: object
+      BlockVersion:
+        required:
+          - bvMajor
+          - bvMinor
+          - bvAlt
+        properties:
+          bvMajor:
+            maximum: 65535
+            minimum: 0
+            type: integer
+          bvMinor:
+            maximum: 65535
+            minimum: 0
+            type: integer
+          bvAlt:
+            maximum: 255
+            minimum: 0
+            type: integer
+        type: object
+      UpdateSeverity:
+        type: enum
+        items: [mandatory, optional]
```

Comment:
Swagger swears that these lines are incorrect; I believe that you wanted to have
```
type: string
enum: [mandatory, optional]
```
instead.

Reply:
I will be honest with you, @flyingleafe : I wrote those on top of my head, so of course they are incorrect, and Swagger is right 😅  Well spotted, I will amend it.

Reply:
I will be honest with you: I wrote those on top of my head, so of course they are incorrect, and Swagger is right 😅 Well spotted, I will amend it.